utils/ansible_galaxy_readme_template.md

```diff
@@ -18,17 +21,17 @@ Dependencies
 
 N/A
 
-Example Playbook
+Example Role Usage
 ----------------
 
 Run `ansible-galaxy install RedHatOfficial.@ROLE_NAME@` to
-download and install the role. Then you can use the following playbook snippet.
+download and install the role. Then, you can use the following playbook snippet to run the Ansible role:
 
     - hosts: all
       roles:
          - { role: RedHatOfficial.@ROLE_NAME@ }
 
-Then first check the playbook using (on the localhost):
+Then, check the playbook using (on the localhost) the following example:
```

Comment:
```suggestion
Next, check the playbook using (on the localhost) the following example:
```

Reply:
(because ``Then,...`` was used in the last paragraph.

Reply:
@shawndwells fixed